### ISSUE TITLE
For all roads include the smoothness value Tag in the data to be filled, as currently in tracks

### ISSUE THREAD

**Issue by jjiglesiasg** [facebook/Rapid#1232]
### Description

Add the Smoothness tag as data to be filled for any kind of ways, not just only for tracks. The classification of a way is NOT dependant of the Smoothness or Rugeness of such way. So adding that value make sense in all roads

**Comment by bhousel** [comment]
Thanks - we can consider it..
Rapid is currently just showing the presets and fields from the https://github.com/openstreetmap/id-tagging-schema project.  

You can make your suggestion there and see whether it gets accepted, but they are very slow to accept things.  (We are also able to override the presets we get from id-tagging-schema, so they are not necessarily a blocker).

Also, it does look like "Smoothness" is available under the "Add field" dropdown, so you can map it, it's just not one of the default fields for the highway presets:
<img width="350" alt="Screenshot 2023-11-27 at 10 47 24 AM" src="https://github.com/facebook/Rapid/assets/38784/f4669991-7ca4-454c-b9db-299c56990159">

**Comment by jjiglesiasg** [comment]
I agree that I could add the Smoothness Tag by myself, but that means also wasting more time when the tag could be a preset and just be filled as needed like access, surface, source or other tags propper of ways.

Please consider. ID is a lost case everybody has a different and sometimes conflicting interest. I tend to believe that ID is managed by the JOSM community that restrict ID to be more efficient and easy to use.
